pkg/ebpf/policies_manager.go

```diff
@@ -0,0 +1,43 @@
+package ebpf
+
+import (
+	"sync"
+
+	"github.com/aquasecurity/tracee/pkg/utils"
+)
+
+// policiesManager is a thread-safe mask of enabled policies
+type policiesManager struct {
+	mutex      sync.Mutex
+	policyMask uint64
+}
```

Comment:
I like the idea of `policesManager`. We'll be able to move to it all bpf logic that's scattered throughout the codebase.

We have a lot of policy settings being done at different tracee initialization steps that need to be brought into a handler like this. Please later take a look at the `TODO:`s inserted by this PR: https://github.com/aquasecurity/tracee/pull/3305/files